frontend/packages/ux-editor/src/components/Preview/Preview.module.css

```
@@ -8,3 +9,9 @@
   width: 100%;
   height: 100%;
 }
+
+.previewLimitationsInfo {
+  padding: 10px;
+  background-color: #efefef;
+  align-content: center;
+}
```

Denne kan forenkles til å bare være margin og så kan vi flytte `background: #efefef ` til å ligge på ` .root {}`. Da kan vi fjerne et element i JSX, se neste kommentar. 😊 
```suggestion
.previewLimitationsInfo {
  margin: 10px;
}
```